Queue/Backend/Sentinel.php

```diff
@@ -36,6 +41,9 @@ protected function connect()
                 $configuredClient = new \Credis_Client($host, $ports[$index], $timeout = 0.5, $persistent = false);
```

Comment:
@snake14 I don't have redis sentinel installed, so couldn't verify it, shouldn't we pass the password here ?
```suggestion

                $configuredClient = new \Credis_Client($host, $ports[$index], $timeout = 0.5, $persistent = false, $db = 0, $this->usePasswordForSentinelInstances && !empty($this->password) ? $this->password: null);
```

Reply:
Okay :+1: